projects/plugins/social/src/js/classic-editor.js

```diff
@@ -0,0 +1,46 @@
+jQuery( function ( $ ) {
+	const state = jetpackSocialClassicEditorInitialState; // eslint-disable-line no-undef -- This is set by wp_localize_script().
```

Comment:
Do you think if this was:
```suggestion
	const state = window.jetpackSocialClassicEditorInitialState ?? {};
```

It would remove the eslint requirement and make sure it doesn't crash if it's not there for some reason?

Reply:
Yup, fixed! [93315c2](https://github.com/Automattic/jetpack/pull/26039/commits/93315c26616a40055366f5afc83249a5858f6da3)